models/jupiter/solana/jupiter_solana_schema.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+
+models:
+  - name: jupiter_solana_dca_volume 
```

Comment:
please apply the unique combo of columns test in this schema file, on the unique keys assigned in the model